examples/nlp/language_modeling/conf/megatron_quantization.yaml

```diff
@@ -20,8 +20,13 @@ trainer:
   precision: bf16 # 16, 32, or bf16
   enable_checkpointing: false
 
+model:
```

Comment:
These are standard parameters for a nemo config hence adding them directly here instead of separately keeping them and then overwriting the model config

Reply:
I believe that megatron_quantization.yaml config was derived based on [megatron_gpt_inference.yaml config](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/conf/megatron_gpt_inference.yaml). I'm not entirely sure but it was the state of things when I first looked into this in ammo repository late '23 (you could ask around).

If that's the case I would suggest let's please keep this config in analogy to the megatron_gpt_inference.yaml config and that config specifies parallelism settings and model file at the top level.

Reply:
I see. 

I only looked at pretraining and SFT-related scripts that use different format as I have updated here. 

For PTQ here we only need `model.tp`, `model.pp` so we just manually overwrite it. 

But for other training based scripts there are many more `model.` parameters that needs to be overwritten so they are directly defined under `model:` section in the yaml so we don't have to manually list all overwrites. I like this approach more as it is more cleaner

If you think this updated version doesn't look good, I can revert PTQ to the previous format. Only thing is that then QAT and PTQ will have different way of defining parameters which may not look good. Thoughts?

Reply:
I see, introducing that `model:` section is fine